src/inmanta/ast/__init__.py

```diff
@@ -587,7 +585,7 @@ class CompilerRuntimeWarning(Warning, RuntimeException):
     """
 
     def __init__(self, stmt: "Optional[Locatable]", msg: str) -> None:
-        Warning.__init__(self)
+        InmantaWarning.__init__(self)
```

Comment:
The class this class inherits from should be updated to `InmantaWarning` as well.